generator/src/main/java/org/corfudb/generator/operations/WriteOperation.java

```diff
@@ -2,8 +2,13 @@
 
 import java.util.UUID;
 
+import ch.qos.logback.classic.Logger;
 import lombok.extern.slf4j.Slf4j;
+import org.corfudb.generator.Correctness;
 import org.corfudb.generator.State;
+import org.corfudb.runtime.CorfuRuntime;
+import org.corfudb.runtime.object.transactions.TransactionalContext;
+import org.slf4j.LoggerFactory;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.slf4j.LoggerFactory'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)